### PR TITLE
fix: Use Memo type to submit/display multi-sig transactions

### PR DESCRIPTION
**Pull request by stanleyjones**
Also see https://github.com/liftedinit/many-js/pull/96

Fixes https://github.com/liftedinit/alberto/issues/109

### CODE REVIEW

File: src/features/transactions/components/send-asset/send-asset.tsx [hunk 1]
```diff
@@ -213,7 +213,7 @@ export function useSendAssetForm({
           to: formValues.to,
           amount: bigIntAmount,
           symbol: asset!.identity,
-          memo: formValues?.memo?.trim(),
+          memo: [formValues?.memo?.trim()],
```
Comment (hansl) [inline src/features/transactions/components/send-asset/send-asset.tsx:216]: Maybe remove it if `memo.trim()` is `""`?

File: package.json [hunk 2]
```diff
@@ -20,7 +20,7 @@
     "@emotion/react": "^11.8.2",
     "@emotion/styled": "^11.8.1",
     "@fontsource/roboto": "^4.5.8",
-    "@liftedinit/many-js": "^0.1.0-alpha.8",
+    "@liftedinit/many-js": "github:stanleyjones/many-js#fix-memo-type",
```
Comment (stanleyjones) [inline package.json:23]: Will change when https://github.com/liftedinit/many-js/pull/96 merges.

File: src/features/transactions/components/txn-list/txn-list-item/multisig-txn-list-item.tsx [hunk 3]
```diff
@@ -204,6 +204,8 @@ export function SubmittedMultisigTxnDetails({
   const { memo, executeAutomatically, threshold, transaction, submitter } =
     (multisigTxnInfoData?.info ?? {}) as MultisigTransactionInfo
 
+  const memoStr = memo && typeof memo[0] === "string" ? memo[0] : ""
```
Comment (stanleyjones) [inline src/features/transactions/components/txn-list/txn-list-item/multisig-txn-list-item.tsx:207]: I could also do a map here. Depends on how many human-readable memos we expect transactions to have.

Comment (hansl) [reply src/features/transactions/components/txn-list/txn-list-item/multisig-txn-list-item.tsx:207]: Just the 1 from the UI for now so I think that's fine.

**Comment by stanleyjones** [comment]
Tested with alberto.liftedinit.dev and seems to be working.

**Pull request merged**